docs/build/nodes.md

```diff
@@ -0,0 +1,39 @@
+# Node configuration
+There are multiple ways to setup, run, and use Ceramic nodes and the optimal setup for you will differ based on you and your users needs.
+
+## Core
+Ceramic core contains the full implementation of the Ceramic protocol and can be used directly though the JavaScript *CeramicApi*. The core implementation can be used in any JavaScript environment, such as your tests, in-browser, or nodejs. When running Ceramic core all streams that are created, loaded, or pinned get verified in the local environment, which is great if you need maximal decentralization and resilience in your application. However, this great power comes with responsibility. In order to run Ceramic Core you need to configure your own IPFS node which supports *dag-jose* and ensure connectivity to the rest of the Ceramic network. Further, a stream created on an instance of Ceramic core will only be available on the network as long as this node remains online. So if your setup for example uses in-browser nodes and your user closes the tab, any stream created by that user will remain unavailable until the user opens the application again. One way to mitigate this is to pin streams on separate long running nodes.
+
+## CLI
+The Ceramic command line interface provides a way to spawn a Ceramic daemon and to interact with this (or remote) daemon though simple commands.
+
+### Ceramic daemon
+The simplest way to run a Ceramic node is to run the `ceramic daemon` command in your console. This command starts a Ceramic node with a standard cofiguration in a few steps. First a properly configured IPFS instance is started, IPFS is then used to create an instance of Ceramic core. Once Ceramic core is running an HTTP server is started that serves the [Ceramic HTTP API](./reference/http-api.md). The node is now ready to accept requests from the local environment or remotely.
+
+By default the daemon runs a bundled IPFS instance. It's also possible to run a separate service for IPFS using the `@ceramicnetwork/ipfs-daemon` npm package. Regardless of how the IPFS instance is managed, if you want your node to be discoverable from the network, make sure to *portforward* the appropriate ports used by IPFS and the Ceramic HTTP API. Also be sure to add your node to the [peerlist](https://github.com/ceramicnetwork/peerlist/blob/main/testnet-clay.json) by submitting a pull request (this is a temporary measure to enhance peer discovery since js-ipfs doesn't support DHT lookups yet).
+
+
+## HTTP Client
+The Ceramic HTTP client is a lighter way of interacting with the Ceramic network. It connects to a remote Ceramic http endpoint to read and write data. Important to note however is the users keys always lives client side and all updates are authored on the client and then sent to the remote http endpoint to write the update to the network. The HTTP client also implements the *CeramicApi* so it can be used interchangeably with Core.
```

Comment:
```suggestion
The Ceramic HTTP client is a lighter way of interacting with the Ceramic network. It connects to a remote Ceramic http endpoint to read and write data. Important to note however is the user's keys always live client side and all updates are authored on the client and then sent to the remote http endpoint to write the update to the network. The HTTP client also implements the *CeramicApi* so it can be used interchangeably with Core.
```